### ISSUE TITLE
[Bug] Visual artefacts at zoom with MVTLayer and stroked=true

### ISSUE THREAD

**Issue by 0xflumedev** [visgl/deck.gl#6966]
### Description

Hi guys. 

Renders ok at zoom 10.
![image](https://user-images.githubusercontent.com/94210181/171561117-9df92dbe-e36d-44cc-89f2-13d24f54aedd.png)
Lots of graphical glitches/visual artefacts at zoom ~11.8
![image](https://user-images.githubusercontent.com/94210181/171561199-dda856b2-5a03-48a7-b405-276aa3e0c1e1.png)
![image](https://user-images.githubusercontent.com/94210181/171561312-ff88d429-fe15-497e-9939-e824fdc47bef.png)
![image](https://user-images.githubusercontent.com/94210181/171561334-64758479-1bc8-4f01-8d8b-5d79c8068129.png)
Reasonably ok again at >12.
![image](https://user-images.githubusercontent.com/94210181/171561243-6fe6d9da-e6be-4a91-880b-95d39831db9c.png)

Can anyone provide any thoughts on if this is something I can fix my end, or is this a glitch in deck.gl?

Deck.gl is exhibiting this on `MVTLayer` with standard hexagonal polygons and `H3HexagonLayer`.

Many thanks

### Flavors

- [X] React
- [ ] Python/Jupyter notebook
- [ ] MapboxLayer
- [ ] GoogleMapsOverlay
- [ ] CartoLayer
- [ ] DeckLayer/DeckRenderer for ArcGIS

### Expected Behavior

No graphical glitches/defects.

### Steps to Reproduce

Instantiation as follows:
```
new MVTLayer({
    id: `hex-layer`,
    data: [hexSourceUrl],
    stroked: true,
    wireframe: false,
    filled: true,
    autoHighlight: false,
    extruded: false,
    getFillColor: (d) => hexToRGBA(HEX_PALETTE[d.properties.d], 255),
    getLineColor: (d) => hexToRGBA(HEX_PALETTE[0], 255),
    lineWidthUnits: "common",
    lineWidthScale: 0.01,
    lineWidthMinPixels: 0,
    lineWidthMaxPixels: 32,
    pickable: false,
    onHover: (info) => console.log("hover", info),
  })
```
`lineWidthScale` is in use in attempt to show only a very thin line at high zoom, and thicker the lower you zoom in.

### Environment

- Framework version:
- Browser:
- OS:

**Comment by 0xflumedev** [comment]
Hello, devs?

**Comment by Pessimistress** [comment]
I can reproduce this in H3HexagonLayer. Please provide a codepen for the MVTLayer issue. You can also try https://deck.gl/docs/developer-guide/tips-and-tricks#z-fighting-and-depth-testing if you only have 2D geometries.

**Comment by Pessimistress** [comment]
Fixed as of v8.8.1

**Comment by 0xflumedev** [comment]
Hi @Pessimistress I meant to write - I tested yesterday your suggestion at `depthTest: false` in that link, it helped improve a lot of artefacts I was seeing.

I really appreciate a code fix on z fighting though, I can't wait to test it! Many thanks again 👍
